besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -898,6 +898,16 @@ void setBannedNodeIds(final List<String> values) {
   private final Integer pendingTxRetentionPeriod =
       TransactionPoolConfiguration.DEFAULT_TX_RETENTION_HOURS;
 
+  @SuppressWarnings({"FieldCanBeFinal", "FieldMayBeFinal"}) // PicoCLI requires non-final.
+  @Option(
+      names = {"--tx-pool-price-bump"},
```

Comment:
Shouldn't we put this command in ExperimentalEIP ?

Reply:
No. This is not related to any EIP.

Reply:
ok :) I thought it was related to the EIP